### DISCUSSION TITLE
questions about patch Seekbar thumbnails for youtube

### DISCUSSION THREAD

**Discussion by yonggamer** [ReVanced/revanced-discussions#1088] [Question] [answered]
why can you patch Seekbar thumbnails on youtube with versions higher than 19.16.39
![png](https://github.com/user-attachments/assets/8f8f88ce-1d5e-4811-bd11-70c536ee7f19)

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Because YouTube removed the player code for the old seekbar thumbnails.

~~If you want old seekbar thumbnails, then patch 19.16.39.~~ Edit: YouTube dropped support for 19.16.39, and now old thumbnails are gone for good.

**Reply by yonggamer** [reply]
but why can you patch youtube versions higher than 19.16.39 when YouTube removed the player code for the old seekbar thumbnails

**Reply by LisoUseInAIKyrios** [reply]
You can patch higher than 19.16.39 because during patching the old seekbar changes are only applied if the target is 19.16.39 or lower.

**Comment by LisoUseInAIKyrios** [comment]
A considerable amount of time was spent trying to restore the missing player thumbnail code to the later targets, but so far all efforts have failed.

Even if that approach did work, it might be too fragile and too complicated to maintain with future targets.

The only viable idea, is to make the full screen thumbnails act more like the old thumbnails. That can be done by resizing the full screen thumbnail view to a smaller size (which fixes the blurry appearance), and to allow seeking while the video continues to play.

**Reply by yonggamer** [reply]
can it be done with a patch?

**Comment by LisoUseInAIKyrios** [comment]
The old seekbar thumbnails are still used for videos playing in the feed, but the code used for the feed seems to be completely different than what is used for the regular player.

**Reply by yonggamer** [reply]
how does it presicely work?

**Reply by LisoUseInAIKyrios** [reply]
Turn on video playback in the feed in the YouTube settings (it's default on), and try it yourself to see.

**Comment by tiklii** [comment]
@LisoUseInAIKyrios 

Is this the same problem as shown in the video here: 
https://www.reddit.com/r/youtube/comments/17hki68/youtube_app_seekingscrubbing_bug/

It doesn't seem to be fixed, right? (Or I'm not able to find the setting for it)

**Reply by LisoUseInAIKyrios** [reply]
The video shows the new seekbar thumbnails.

Since YouTube dropped support for 19.16.39 there no longer is anyway to use the old thumbnails.  What that video shows is correct and there is no way to change it.
